snappy_server/native/invectedgamelogic/src/game_logic.rs

```diff
@@ -0,0 +1,143 @@
+extern crate serde;
+extern crate serde_json;
+
+use std::sync::Arc;
+use std::collections::HashMap;
+use std::collections::hash_map::Entry;
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct Player {
+    name: String,
```

Comment:
Is de naam de unique identifier van de player?

Reply:
Op dit moment wordt de speler inderdaad uniek geidentificeerd binnen een spel op basis van zijn naam; dit was in de Unity-versie van het spel ook het geval.

Mogelijk is dit wel iets om inderdaad te veranderen, want dan kunnen we een vector gebruiken als lijst van spelers ipv een hashmap. Vast een vorm van premature optimization, maar geen slecht idee :+1:!

Reply:
Aan de display kant kan ik voor nu ook gewoon een hashmap van players bijhouden, en die dan updaten gebaseerd op de naam. Dan pas ik mijn kant gewoon ook aan wanneer hier ook een ID ontstaat.